README.md

```diff
@@ -93,9 +93,12 @@ cd manifests/overlays/dev && kustomize edit set image quay.io/operate-first/serv
 {oc,kubectl} apply -k manifests/overlays/dev
 ```
 
-### Docker image
-The image can be built using:
+### S2I image
+This repository uses s2i to create images. To create image run script located in `packages/backend`:
 ```sh
-yarn build-image
+./packages/backend/build-image.sh
+```
+This will generate an image called `service-catalog`, which then can be run with:
+```sh
+podman run -it --network host --env "backend_secret=$(node -p 'require("crypto").randomBytes(24).toString("base64")')" service-catalog:latest
```

Comment:
```suggestion
podman run -it -p 7007:7007 --env "backend_secret=$(node -p 'require("crypto").randomBytes(24).toString("base64")')" service-catalog:latest
```

Port 7007 works just fine for me...

Reply:
After reinstall of podman it started working no idea what was wrong :man_shrugging:. Resolved